.github/workflows/test_frameworks.yml

```
@@ -286,7 +286,7 @@ jobs:
         if: needs.needs-run.outputs.outcome == 'success'
         run: PYTHONPATH=../ddtrace/tests/debugging/exploration/ ddtrace-run pytest -p no:warnings tests
       - name: Debugger exploration results
-        if: always()
+        if: needs.needs-run.outputs.outcome == 'success'
         run: cat debugger-expl.txt || true
```

Do we still need the `true` condition? 
```suggestion
        run: cat debugger-expl.txt
```